testbed/testbed/child_process_collector.go

```diff
@@ -172,7 +172,11 @@ func (cp *childProcessCollector) Start(params StartParams) error {
 	exePath := expandExeFileName(cp.AgentExePath)
 	exePath, err := filepath.Abs(exePath)
 	if err != nil {
-		return err
+		panic(err)
+	}
+
+	if _, err = os.Stat(exePath); err != nil {
+		panic(err)
```

Comment:
Can we make this more informative by wrapping it in `fmt.Errorf` so that we can add some additional human context?

Reply:
instead of `fmt.Errorf` i did something like `panic(fmt.Sprintf(....))`

Reply:
More human readable is better :D Don't mind how it gets there.